roles/download-k8s/tasks/main.yml

```diff
@@ -17,3 +17,9 @@
   with_items: "{{ k8s_tar_bundles }}"
   retries: 3
   delay: 5
+
+- name: Import container images
+  shell: |
+    export ARCH=ppc64le
+    find /usr/local/bin -name "*.tar" -print | xargs -L 1 ctr -n k8s.io images import
```

Comment:
this logic goes for a toss if we have some other tars in that directory, lets list iterate over only the images required to stand up the cluster